lib/msal-common/src/telemetry/server/ServerTelemetryManager.ts

```diff
@@ -35,36 +37,41 @@ export class ServerTelemetryManager {
         return [SERVER_TELEM_CONSTANTS.SCHEMA_VERSION, request, platformFields].join(SERVER_TELEM_CONSTANTS.CATEGORY_SEPARATOR);
     }
 
-    // API to add MSER Telemetry for the last failed request
+    /**
+     * API to add MSER Telemetry for the last failed request
+     */
     generateLastRequestHeaderValue(): string {
         const lastRequests = this.getLastRequests();
         
-        const failedRequests = lastRequests.failedRequests.join(SERVER_TELEM_CONSTANTS.VALUE_SEPARATOR);
-        const errors = lastRequests.errors.join(SERVER_TELEM_CONSTANTS.VALUE_SEPARATOR);
-        const platformFields = lastRequests.errorCount;
+        const maxErrors = ServerTelemetryManager.maxErrorsToSend(lastRequests);
+        const failedRequests = lastRequests.failedRequests.slice(0, 2*maxErrors).join(SERVER_TELEM_CONSTANTS.VALUE_SEPARATOR);
+        const errors = lastRequests.errors.slice(0, maxErrors).join(SERVER_TELEM_CONSTANTS.VALUE_SEPARATOR);
+        const errorCount = lastRequests.errors.length;
+
+        // Indicate whether this header contains all data or partial data
+        const overflow = maxErrors < errorCount ? "1" : "0";
```

Comment:
Are 1 and 0 true/false here? Adding a constant for both might help with context, as these are magic strings, technically.

Reply:
Done